proto/buf/alpha/registry/v1alpha1/doc.proto

```diff
@@ -473,6 +473,7 @@ message LiteralValue {
     ArrayLiteral array_value = 8;
     MessageLiteral message_value = 9;
   }
+  string comments = 10;
```

Comment:
You renamed this comments but not everything else - update everything to be consistent

Reply:
👍 got it, my bad I thought we only wanted to do it here. Fixed in latest commit.

Reply:
Uh, do we use JSON between the BSR web UI and the BSR server? If so, this is not a backwards compatible change and will break the current functionality for comments (the code will be looking for a `description` property in the JSON payload).

I guess we don't have `buf breaking` enabled for this repo, since nothing complained about this?

Reply:
Only the UI uses this, and so assumedly @gilwong00 will update the BSR anyways. We don't check for breaking changes on alpha APIs.

Reply:
Yea, i have a PR to make these changes in the BSR so we shouldnt be affected for to long